tools/mmp/driver.cs

```diff
@@ -1251,6 +1251,7 @@ static int Compile ()
 					args.Append ("-fobjc-runtime=macosx ");
 				if (!embed_mono)
 					args.Append ("-DDYNAMIC_MONO_RUNTIME ");
+				args.Append ("-Wno-unguarded-availability-new ");
```

Comment:
does older `clang` versions support it ?
or are we trading this warning for another ?